data/json/items/comestibles/mutagen.json

```diff
@@ -21,182 +21,83 @@
     },
     {
         "id" : "mutagen_beast",
+        "copy-from" : "mutagen",
         "type" : "COMESTIBLE",
-        "comestible_type" : "DRINK",
         "name" : "beast mutagen",
         "description" : "A rare substance of uncertain origins.  Causes you to mutate.",
-        "category" : "mutagen",
-        "weight" : 250,
-        "volume" : 1,
-        "price" : 100000,
-        "phase" : "liquid",
-        "container" : "flask_glass",
-        "symbol" : "~",
-        "color" : "light_green",
-        "healthy" : -2,
         "addiction_potential" : 2,
```

Comment:
It's my bad on that. I didn't see the above definition for `mutagen` and thought it was still to be implemented